pkgs/os-specific/linux/systemd/default.nix

```
@@ -129,6 +133,7 @@
 assert withImportd -> withCompression;
 assert withCoredump -> withCompression;
 assert withHomed -> withCryptsetup;
+assert withHomed -> withPam;
```

I have a question on this one: what would be the preferred way to do something like that? Would it be what I have done in the change or something like this instead:
```suggestion
assert withHomed -> withCryptsetup && withPam;
```